src/hearings/containers/request-hearing/hearing-change-reason/hearing-change-reason.component.ts

```diff
@@ -99,12 +100,13 @@ export class HearingChangeReasonComponent extends RequestHearingPageFlow impleme
           key: reason.value.key,
           value_en: reason.value.value_en,
           value_cy: reason.value.value_cy,
-          hintText_EN: reason.value.hintText_EN,
-          hintTextCY: reason.value.hintTextCY,
-          order: reason.value.order,
-          parentKey: reason.value.parentKey,
+          hint_text_en: reason.value.hint_text_en,
```

Comment:
is it worth trying to use {...control.value} instead of each key assignment as the source and destination object holding the same keys

Reply:
thanks that is a good point, absolutely we can do that, however I will keep the existing code for data transformation to avoid retest the whole journey.